core/src/main/java/io/aiven/klaw/helpers/KlawResourceUtils.java

```diff
@@ -0,0 +1,24 @@
+package io.aiven.klaw.helpers;
+
+import io.aiven.klaw.dao.Env;
+import io.aiven.klaw.model.response.EnvIdInfo;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+public class KlawResourceUtils {
```

Comment:
how abt using commonutils ? we have similar methods there

Reply:
CommonUtils is instantiated and also makes calls out to the db etc, my thoughts for this one were more like a StringUtils or CollectionUtils. A bunch of static methods that don't need to be mocked but allow us to merge a lot of common processing into one location.